.github/workflows/auto-close-issues.yml

```
@@ -27,7 +27,8 @@ jobs:
           return pr.data.merged
 
     - name: Get Linked Issues
-      if: steps.check-merge.outputs.result == 'true'
+      if: >
```

actionlint (1.7.7)

property "get-issues" is not defined in object type {check-merge: {conclusion: string; outcome: string; outputs: object}}

(expression)